web/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import { Column, Columns, Box, Link} from 'fannypack';
+import logo from '../../images/logo.png';
+import "./style.css";
+
+class Footer extends Component {
+    render(){
+        return(
+          <Columns isOneLine>
+          <Column spread={12}>
+          <Box backgroundColor="transparent">
+          <Link href="https://opencharlotte.org/"><img className="ocbLogo"src={logo} alt="Logo" /></Link>
```

Comment:
We should be styling here using emotion.


```js
import styled from 'emotion'

const Logo = styled('img')`
    height: 4rem;
    width: auto;
    margin-left: 44%;
`
```

Then we can use our styled component rather than the plain element we are using:

```jsx
 <Link href="https://opencharlotte.org/"><Logo src={logo} alt="Logo" /></Link>
```

Reply:
The way that the image is styled actually leads to some interesting behavior when the link is clicked

![image](https://user-images.githubusercontent.com/6907797/54066694-83cb0900-4202-11e9-9f41-3f2019a0bf45.png)

A better way to center are to use a `display: flex;` with a `justify-content: center;` on the parent component (in this case that would be our Box).  Fortunately for us, Box already exposes the ability to override css as props:

```jsx
  <Box backgroundColor="transparent" display="flex" justifyContent="center">
    <Link href="https://opencharlotte.org/">
      <img className={logoClass} src={logo} alt="Logo" />
    </Link>
  </Box>
```  

Notice that we just add the **display** and **justifyContent** props.  This leads to a bit cleaner UI when the link is clicked.

![image](https://user-images.githubusercontent.com/6907797/54066729-431fbf80-4203-11e9-9d18-4b312dfcb65c.png)


If we were doing this using emotion, it would look something like the following:

```jsx
const logoBoxClass = css`
    background-color: transparent;
    display: flex;
    justify-content: flex-end;
`

//elsewhere

  <Box className={logoBoxClass}>
    <Link href="https://opencharlotte.org/">
      <img className={logoClass} src={logo} alt="Logo" />
    </Link>
  </Box>
```